live/lekce3/10.py

```diff
@@ -0,0 +1,37 @@
+import random
+from pprint import pprint 
+soubor = open("karty.txt","r")
+deck = [card for card in soubor]
+soubor.close()
+
+
+# print("zaprasene " + str(deck))
+# print("zaprasene {}".format(deck))
+pprint(f"zaprasene: {deck}")
+
+deck_strip = [card.strip() for card in deck]
+pprint(f"odprasene: {deck_strip}")
+
+
+# zamichat a vybrat hodnoty
+random.shuffle(deck_strip)
+
+pprint(f"zamichano: {deck_strip}")
+
+vybrane_karty = deck_strip[:4]
```

Comment:
Mozna bych sem precijen pridal komentar proc staci vzit prvni 4, kdyz to nekdo uvidi bez toho vysvetleni, tak by to nemuselo byt hned jasne.